### PR TITLE
refactor: rename to reqd_confs

### PR DESCRIPTION
**Pull request by prestwich**
## Motivation

Clarify naming on pending transaction and associated structs

## Solution

rename `confirmations` to `reqd_confs`

## PR Checklist

- [ ] Added Tests
- [ ] Added Documentation
- [ ] Breaking changes

### CODE REVIEW

**Review by DaniPopes — APPROVED** [review]

File: crates/provider/src/heart.rs [hunk 1]
```diff
@@ -108,18 +108,18 @@ impl<'a, N: Network, T: Transport + Clone> PendingTransactionBuilder<'a, N, T> {
     }
 
     /// Returns the number of confirmations to wait for.
-    pub const fn confirmations(&self) -> u64 {
-        self.config.confirmations()
+    pub const fn reqd_confs(&self) -> u64 {
```
Comment (DaniPopes) [inline crates/provider/src/heart.rs:111]: maybe `#[doc(alias)]` on at least one of these for discoverability?

**Review by DaniPopes — COMMENTED** [review]
Needs a few more updates

File: crates/provider/src/heart.rs (hunk 1 above)
Comment (prestwich) [reply crates/provider/src/heart.rs:111]: I'll go ahead and alias them all 👍

File: crates/provider/src/heart.rs [hunk 2]
```diff
@@ -108,18 +108,21 @@ impl<'a, N: Network, T: Transport + Clone> PendingTransactionBuilder<'a, N, T> {
     }
 
     /// Returns the number of confirmations to wait for.
```
Comment (mattsse) [inline crates/provider/src/heart.rs:110]:
```suggestion
    /// Returns the number of required confirmations to wait for.
```
or is this requested?

**Review by onbjerg — APPROVED** [review]

**Review by mattsse — APPROVED** [review]

**Pull request merged**